example/sofast_fixed/example_find_blobs_in_image.py

```diff
@@ -1,32 +1,35 @@
 """Example script that finds blobs in image and saves annotated image.
 """
 
-import os
-from os.path import join, dirname, exists
+from os.path import join, dirname
 
 import cv2 as cv
 
 from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
 from opencsp.app.sofast.lib.MeasurementSofastFixed import MeasurementSofastFixed
 from opencsp.app.sofast.lib.image_processing import detect_blobs_annotate
+import opencsp.common.lib.tool.file_tools as ft
+import opencsp.common.lib.tool.log_tools as lt
 
 
 def example_find_blobs_in_image():
-    """Finds blobs in image, annotates, and saves image"""
-    file_meas = join(
-        opencsp_code_dir(),
-        '../../sample_data/deflectometry/sandia_lab/fixed_pattern/measurement_screen_square_width3_space6.h5',
-    )
-    file_save = join(dirname(__file__), 'data/output/blob_detection/image_with_detected_blobs.png')
+    """Finds blobs in image, annotates image, and saves"""
+    # General Setup
+    dir_save = join(dirname(__file__), 'data/output/find_blobs_in_image')
+    ft.create_directories_if_necessary(dir_save)
 
-    if not exists(dirname(file_save)):
-        os.makedirs(dirname(file_save))
+    lt.logger(join(dir_save, 'log.txt'), lt.log.INFO)
 
-    # Load image
+    # Load image from measurement file
+    file_meas = join(opencsp_code_dir(), 'test/data/sofast_fixed/data_measurement/measurement_facet.h5')
     measurement = MeasurementSofastFixed.load_from_hdf(file_meas)
     image = measurement.image
 
-    # Detect blobs
+    # Load image from existing image file
+    # image = cv2.imread(file_jpg, cv2.IMREAD_GRAYSCALE)
+    # image = imageio.imread(file_png)
```

Comment:
I like that you have an alternative example of how to get the image data! I just have two quick comments about it:
1. Does this second line ("imageio.read") do anything?
2. You should probably say that this is an alternative way to get image data in the comment. Without that, someone might mistake this for old code in the future and delete the comment.

Reply:
Yeah, that could be more clear. I'll change this to only use one method (OpenCV, not OpenCV and imageio) and add more documentation.